.github/workflows/template.python-wheels-ci.yml

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Python Wheels CI
+
+on:
+  workflow_call:
+
+permissions: read-all
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+      - name: Build wheels
+        run: |
+          cd flink-python
+          bash dev/build-wheels.sh
+      - name: Tar artifact
+        run: tar -cvf python-wheel.tar flink-python/dist
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: wheel_${{ runner.os }}_${{ github.sha }}_${{ github.run_number }}
```

Comment:
This is slightly different than the Azure pipelines build. There `Agent.JobName` is used.

```
- task: PublishPipelineArtifact@0
        inputs:
          artifactName: 'wheel_$(Agent.OS)_$(Agent.JobName)'
          targetPath: 'flink-python/dist'
```

Could you please suggest how could we make them similar?

Reply:
What are you referring to with "them"? The `name`, I guess?

I'm wondering whether we should rely on what's provided in [apache/flink:.github/workflows/template.flink-ci.yml:106}(https://github.com/apache/flink/blob/master/.github/workflows/template.flink-ci.yml#L106) with the stringified workflow name. WDYT?

Reply:
Hey @XComp,

I meant to make the python-wheels-ci GitHub workflow and Azure pipelines.

Here the main concern is to keep the Python Wheels artifact names similar. Since the Azure pipelines used `Agent.JobName` it would be fine to use the stringrified workflow name, with `wheel` prefixed.

Would that be good option?